Mathlib/Data/List/Basic.lean

```diff
@@ -2069,6 +2069,9 @@ theorem tail_drop (l : List α) (n : ℕ) : (l.drop n).tail = l.drop (n + 1) :=
     · simp [hl]
 #align list.tail_drop List.tail_drop
 
+theorem drop_tail (l : List α) (n : ℕ) : l.tail.drop n = l.drop (n + 1) := by
```

Comment:
This could be a useful simp lemma

Reply:
`List.tail_drop` isn't marked as simp either. I'll try adding them both